1460.make-two-arrays-equal-by-reversing-subarrays.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/make-two-arrays-equal-by-reversing-subarrays/
+ */
+
+/**
+ * @param {number[]} target
+ * @param {number[]} arr
+ * @return {boolean}
+ */
+var canBeEqual = function(target, arr) {
+    
+    const sortTarget = target.sort();
+    const sortArr = arr.sort();
+    
+    return sortTarget.toString() === sortArr.toString();
```

Comment:
```suggestion
    const sortedTarget = target.sort();
    const sortedArr = arr.sort();
    
    return sortedTarget.toString() === sortedArr.toString();
```
These names seem more self-descriptive